docs/source/error-codes/self-service/index.rst

```diff
@@ -380,126 +380,150 @@ Ledger API
 The table below outlines generic gRPC status code changes pertaining to the Ledger API
 and apply to all ledger backends. For changes specific to a ledger backend, check the next subsections.
 
-+-----------------------------------------------+-----------------------------------+----------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|Service endpoint                               |gRPC status code (before SDK 1.18) |gRPC status code (since SDK 1.18) |Remarks                                                                                                                  |Ledger API error code ID                     |
```

Comment:
Went from horizontal
```
gRPC status code (before SDK 1.18)
```
to vertical
```
gRPC status code 
(before SDK 1.18)
```
to make this table a bit narrower.

Reply:
:+1: